zeekscript/cli.py

```diff
@@ -173,3 +178,6 @@ def add_parse_cmd(parser):
     parser.add_argument(
         'script', metavar='FILE', nargs='?',
         help='Zeek script to parse. ' + FILE_HELP)
+    parser.add_argument(
+        '--quiet', '-q', action='store_true',
+        help='supress output and just return success or failure')
```

Comment:
Minor typo here, I can fix that when merging. Or, feel free to push an update.

Reply:
oops